src/desdemona/catalogs/sample_catalog.clj

```diff
@@ -1,5 +1,4 @@
-(ns desdemona.catalogs.sample-catalog
-  (:require [desdemona.functions.sample-functions :refer [transform-segment-shape prepare-rows]]))
+(ns desdemona.catalogs.sample-catalog)
```

Comment:
Just confirmed with the onyx folks. Something somewhere does need to require it, but that doesn't need to be in the catalog, which is kinda what I guessed.